src/coreclr/tools/Common/Compiler/ProcessLinkerXmlBase.cs

```diff
@@ -10,10 +10,17 @@
 using System.Reflection.Metadata;
 using System.Text;
 using System.Text.RegularExpressions;
+#if !READYTORUN
```

Comment:
I actually enabled it for NativeAOT https://github.com/dotnet/runtime/blob/main/src/coreclr/tools/aot/.editorconfig#L5, so it would break the crossgren build